src/blenderbim/blenderbim/bim/module/unit/operator.py

```diff
@@ -248,8 +248,7 @@ def import_attributes(self, name, prop, data, context):
             new.name = name
             new.is_null = data[name] is None
             new.is_optional = False
-            new.data_type = "string"
-            new.string_value = json.dumps([e for e in IfcStore.get_file().by_id(data["id"]).Dimensions])
+            new.set_value(json.dumps([e for e in IfcStore.get_file().by_id(data["id"]).Dimensions]))
```

Comment:
Does this actually work? I think it won't. Note that this is a callback - we are explicitly overriding the data_type to string. If you delete it then it won't know what to set value.

Reply:
Unless `json.dumps` doesn't always return a `str` it should get detected and the `data_type` assigned in `set_value` :

https://github.com/IfcOpenShell/IfcOpenShell/blob/v0.6.0/src/blenderbim/blenderbim/bim/prop.py#L203-L215

But no problem rather safe than sorry then :)